appeals/api/src/server/endpoints/appellant-cases/appellant-cases.controller.js

```diff
@@ -93,7 +93,7 @@ const updateAppellantCaseById = async (req, res) => {
 					siteAreaSquareMetres,
 					applicationDate,
 					applicationDecisionDate,
-					changedDevelopmentDescription: !developmentDescription.isCorrect,
+					changedDevelopmentDescription: developmentDescription.isCorrect,
```

Comment:
It needs to be the inverse of isCorrect. If the development description was change it wasn't correct: changedDescription == !isCorrect

Reply:
Same as the other one, doing this inverts the answer